tools/cpp/logger/main.cpp

```diff
@@ -207,7 +208,7 @@ struct Args
         }
 
         if (rotate > 0 && auto_increment) {
-            cerr << "ERROR.  --increment and --rotate can't both be used" << endl;
+            cerr << "ERROR.  --increment and --rotate can't both be used. Note that if you don't want --increment, you must specify a log filename." << endl;
```

Comment:
Can we make sure to adhere to max line length here? Run `zcm-logger -h` to see what I mean